packages/edge-preview-authenticated-proxy/src/index.ts

```diff
@@ -384,6 +400,16 @@ export default {
 					}
 				);
 			}
+		} finally {
+			ctx.waitUntil(
+				fetch("https://workers-logging.cfdata.org/prometheus", {
```

Comment:
What happens if this `fetch()` fails?

Reply:
Oh I guess nothing since it does not block the response, so it can have no impact on the client request.